lib/evss/documents_service.rb

```diff
@@ -26,5 +28,12 @@ def upload(file_body, document_data)
     def self.breakers_service
       BaseService.create_breakers_service(name: 'EVSS/Documents', url: BASE_URL)
     end
+
+    private
+
+    verify_readability_of(file_body)
+      file = PDF::Reader.new(file_body)
```

Comment:
I don't know much about the performance of PDF::Reader but wondering if checking *every* doc is the right choice -- if the header says "application/pdf" can you trust it and then check?